### PR TITLE
Homepage Links to Populated Map or HR Events Page

### PR DESCRIPTION
**Pull request by asedarski**
@ezzy1337 Review?

This PR links the user to the Map. If the Explore Schools button is clicked, it will prepopulate the map with the school data. If the Explore Crime button is clicked, it will prepopulate the map with crime data. 

The "People" section has been updated to have a button "Explore Events" which links the user to the http://hamptonroads.eventful.com/events page.

Issue #20 
Issue #21 
Issue #54

### CODE REVIEW

File: public/js/map.js [hunk 1]
```diff
@@ -62,13 +62,23 @@ function initAutocomplete() {
...
+            crimeheatmap = new google.maps.visualization.HeatmapLayer({
+                data: pointArray,
+                maxIntensity: 20,
+                dissipating: true
             });
             crimeheatmap.setMap(map);
+            google.maps.event.addListenerOnce(map, 'tilesloaded', function(){
+                var selection = getUrlParameter('selection');
+
+                if (selection == 'crime') {
```
Comment (hop-along-polly) [inline public/js/map.js]: Since javascript has it we should use the `===` comparison.

File: public/js/map.js [hunk 2]
```diff
@@ -62,13 +62,23 @@ function initAutocomplete() {
...
+                    populateCrime();
+                }
+                if (selection == 'school') {
```
Comment (hop-along-polly) [inline public/js/map.js]: `===`

**Review by hop-along-polly — APPROVED** [review]
Looks good!

**Comment by asedarski** [comment]
Updated

**Pull request merged**